Pod/Classes/DatabaseConnection.swift

```diff
@@ -12,6 +12,8 @@ import sqlite3
 /** Valid SQLite types are marked using the 'SQLiteValue' protocol */
 public protocol SQLiteValue {}
 
+extension NSArray: SQLiteValue {}
+extension NSDictionary: SQLiteValue {}
```

Comment:
Yes, i know. Maybe SwiftyDB should define its own protocol for valid value types. Would make it less dependent on TinySQLite, and make future changes easier.

Great pull request, by the way!